src/applications/mhv-medical-records/containers/LabsAndTests.jsx

```diff
@@ -86,20 +92,30 @@ const LabsAndTests = () => {
         listCurrentAsOf={labsAndTestsCurrentAsOf}
         initialFhirLoad={refresh.initialFhirLoad}
       >
-        <NewRecordsIndicator
-          refreshState={refresh}
-          extractType={[refreshExtractTypes.CHEM_HEM, refreshExtractTypes.VPR]}
-          newRecordsFound={
-            Array.isArray(labsAndTests) &&
-            Array.isArray(updatedRecordList) &&
-            labsAndTests.length !== updatedRecordList.length
-          }
-          reloadFunction={() => {
-            dispatch(reloadRecords());
-          }}
+        {!isAcceleratingLabsAndTests && (
+          <NewRecordsIndicator
```

Comment:
QUESTION: This just _hides_  the PHR refresh, but doesn't stop it from happening. Should we stop the refresh from happening?